### PR TITLE
DBZ-2620 Adding open graph tags

### PR DESCRIPTION
**Pull request by uidoyen**

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -6,13 +6,43 @@
...
   <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+
+  <meta content="{{ site.title }}" property="og:site_name">
+  {% if page.title %}
+  <meta content="{{ page.title }}" property="og:title">
+  {% else %}
+  <meta content="{{ site.title }}" property="og:title">
+  {% endif %}
+  {% if page.image %}
+  <meta content="{{ site.url }}{{ site.url }}{{ post.image }}" property="og:image">
```
Comment (gunnarmorling) [inline _layouts/base.html:17]: page.image or post.image?

**Comment by gunnarmorling** [comment]
Nice one; can you push this to the preview branch (see README) so we can take a look?

File: _layouts/base.html (hunk 1 above)
Comment (uidoyen) [reply _layouts/base.html:17]: Actually `post.image`

Comment (gunnarmorling) [reply _layouts/base.html:17]: Ok, but on the line before you query for page.image. Is that intended?

Comment (uidoyen) [reply _layouts/base.html:17]: Actually it was a typo since we are adding social sharing icons for blogs, it should be `post.image`

**Comment by gunnarmorling** [comment]
@uidoyen, kinda forgot about this one; is it good to go from your PoV?

**Comment by uidoyen** [comment]
> @uidoyen, kinda forgot about this one; is it good to go from your PoV?
Yes, I think it's good to go.

File: _layouts/base.html [hunk 2]
```diff
@@ -6,13 +6,43 @@
...
+  <meta name="twitter:creator" content="@{{ page.author }}">
+  {% else %}
+  <meta name="twitter:creator" content="@debezium">
+  {% endif %}
+  {% if page.summary %}
+  <meta name="twitter:description" content="{{ page.summary }}">
+  {% else %}
+  <meta name="twitter:description" content="{{ site.description }}">
+  {% endif %}
+  <meta name="twitter:card"  content="summary_large_image">
```
Comment (gunnarmorling) [inline _layouts/base.html:35]: What is `summary_large_image`? It's not a variable, I suppose? Should it be removed?

Comment (uidoyen) [reply _layouts/base.html:35]: `summary_large_image` is for Twitter card view (https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image). Yes, It's not a variable of Jekyll. Twitter picks up large images from post as far as I am aware of.

Comment (gunnarmorling) [reply _layouts/base.html:35]: Oh, I see. Ok, let's give it a try then. Thanks!

**Pull request merged**